kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-api/kie-wb-common-stunner-client-api/src/main/java/org/kie/workbench/common/stunner/core/client/canvas/Canvas.java

```diff
@@ -95,4 +95,10 @@
      * Destroy whatever canvas state present, it will be no longer used.
      */
     void destroy();
+
+    /**
+     * Verify if the event handles of the canvas are enabled or not.
+     * @return true if it is enabled, false otherwise
+     */
+    boolean isEventHandlesEnabled();
```

Comment:
hey @danielzhe 
The only thing I woulod  prefer not having is this new method in the Canvas API. I would try to keep the api as much clean as possible, and events are not really related to canvas itself, they're different things.
So if it's not hard, could you please try to just add this new `isEventHandlesEnabled` method in some abstraction or top class or interface? eg: AbstractCanvas, LienzoCanvas... It means you'll have to ensure the callers will not complain, but I think all `XXXSessionCommand` classes are already having an AbstractCanvas as argument, so maybe that's just so easy.
Could you please take a quick look and lemme know? Otherise I will not block this PR anyway, but would be nice not introducing this method in the interface now if possible.
Thanks in advance!

Reply:
Thank you, @romartin ! I did the change you proposed. I gonna improve the test coverage now. Thank you again for your review and advice!